tests/test_core.py

```diff
@@ -6989,6 +6989,20 @@ def test_ccall(self):
     if self.maybe_closure():
       self.do_core_test('test_ccall.cpp')
 
+  def test_ccall_cwrap_fast_path(self):
+    self.emcc_args.append('-Wno-return-stack-address')
+    self.set_setting('EXPORTED_RUNTIME_METHODS', ['ccall', 'cwrap'])
+    self.set_setting('WASM_ASYNC_COMPILATION', 0)
+    self.set_setting('ASSERTIONS', 0)
+    create_file('post.js', '''
+      var printBool = Module['cwrap']('print_bool', null, ['boolean']);
+      out(Module['_print_bool'] === printBool); // the function should be the exact raw function in the module rather than a wrapped one
+      ''')
+    self.emcc_args += ['--post-js', 'post.js']
+
+    self.set_setting('EXPORTED_FUNCTIONS', ['_print_bool'])
+    self.do_core_test('test_ccall.cpp', out_suffix='_cwrap_fast_path')
```

Comment:
You could just to `self.do_runf(test_file('core/test_ccall.cpp`), 'true')` to avoiding adding the new out file.

Reply:
Thanks for the suggestion 😄 , I've applied this approached now.